src/debug/debug_impl.ts

```diff
@@ -860,7 +860,7 @@ class ThreadManager {
 
 	removeStoredIds(ids: number[]) {
 		for (let id of ids) {
-			delete this.storeData[id];
```

Comment:
I suspect so; though how much of how often, I don't know. I'll try and get this merged soon anyway; I've wanted to use these TS options for a while but never got round to completely getting it compiling again!